creator/ingest_runs/manifests.py

```diff
@@ -0,0 +1,295 @@
+import boto3
+import os
+import pandas as pd
+
+from d3b_utils.aws_bucket_contents import fetch_bucket_obj_info
+from kf_lib_data_ingest.app import settings
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.common.io import read_df
+from kf_lib_data_ingest.etl.load.load import LoadStage
+from kf_utils.dataservice.scrape import yield_entities
+
+from urllib.parse import urlparse
+
+DATASERVICE_URL = "https://kf-api-dataservice.kidsfirstdrc.org/"
```

Comment:
@gsantia I think this URL will be different depending on the environment we're deploying to. So I think you will want to add this variable to each of the dev, testing, production settings files. The value should be read in from an environment variable. If not set the default should point to `LOCALHOST`. Don't want to accidentally modify prod

This shouldn't matter for your unit tests since you're mocking data service responses out. But it will matter later for deployment and also if we want to do integration tests with our data service deployments.

Later on we will need to update the deployment environment variables on S3 for each study creator deployment environment. I think when we deploy to the dev environment the URL would be set to localhost. When we deploy to QA we'd point the environment variable to QA data service and when we deploy to prod we'd point the var to production data service.